src/components/data/PowerBIReport/index.tsx

```diff
@@ -336,9 +336,8 @@ const PowerBIReport: React.FC<PowerBIProps> = ({ reportId, filters, hasContext }
             }
         }
     }, [embedRef, accessToken, isFetching]);
-
-    if (powerBIError || fusionError) {
-        //Only handling selected errors from Power BI. As you migh get errors that can be ignored.
+    if (powerBIError || fusionError?.fusionError) {
```

Comment:
Are you sure about this ? 
There is a new access check here. If fusionError is set, the error should be valid.

Reply:
see checkContextAccess. 
If you are testing with a report that has RLS config, this runs.
If you dont have access to the external ID, you might end up with no access.

Reply:
For me, that is not the case. When I tried to access the analytics reports the `fusionError` was set but `fusionError.fusionError` and `fusionError.status` was undefined